test/hotspot/jtreg/serviceability/sa/sadebugd/DebugdUtils.java

```diff
@@ -35,13 +35,27 @@
 
     private final String id;
 
+    private int registryPort;
+
+    private boolean disableRegistry;
+
```

Comment:
I'd suggest removing the existing empty lines rather than follow the pattern of adding them.